src/resources/Microsoft.Authorization/policySetDefinitions/Enforce-EncryptTransit_20240509.json

```diff
@@ -5,13 +5,15 @@
     "scope": null,
     "properties": {
         "policyType": "Custom",
-        "displayName": "Deny or Deploy and append TLS requirements and SSL enforcement on resources without Encryption in transit",
-        "description": "Choose either Deploy if not exist and append in combination with audit or Select Deny in the Policy effect. Deny polices shift left. Deploy if not exist and append enforce but can be changed, and because missing existence condition require then the combination of Audit. ",
+        "displayName": "[Deprecated]: Deny or Deploy and append TLS requirements and SSL enforcement on resources without Encryption in transit",
+        "description": "Choose either Deploy if not exist and append in combination with audit or Select Deny in the Policy effect. Deny polices shift left. Deploy if not exist and append enforce but can be changed, and because missing existence condition require then the combination of Audit. Superseded by https://www.azadvertizer.net/azpolicyinitiativesadvertizer/Enforce-EncryptTransit_20241211.html ",
         "metadata": {
-            "version": "1.0.0",
+            "version": "1.2.0-deprecated",
```

Comment:
should this be `1.2.0`? surley it stays at `1.0.0` and the new version goes to `1.2.0`?

Reply:
Oversight, thanks for pointing out this copy and paste issue. Corrected.